ui/shared/components/panel/variants/selectors.test.js

```diff
@@ -67,7 +79,7 @@ test('getPairedFilteredSavedVariants', () => {
 
 test('getVisibleSortedSavedVariants', () => {
   const savedVariants = getVisibleSortedSavedVariants(
-    STATE_WITH_2_FAMILIES, { match: { params:  {} } }
+    STATE_WITH_2_FAMILIES, { project: { projectGuid: 'R0237_1000_genomes_demo' }, match: { params:  {} } },
```

Comment:
we should also be testing without the project and with the new state field to check that that works correctly too